Cargo.toml

```diff
@@ -43,18 +43,18 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 timed = { git = "https://github.com/nikkolasg/timed-rs" }
 timed-core = { git = "https://github.com/nikkolasg/timed-rs" }
 
-[profile.dev]
-lto = "thin"
-# We are running our tests with optimizations turned on to make them faster.
-# Please turn optimizations off, when you want accurate stack traces for debugging.
-opt-level = 2
+# Test must run fast, but still provides stacktraces.
+[profile.test]
```

Comment:
Can we keep a `dev` profile as well? It would be useful if we want to run binaries and/or examples (e.g., the bench in `bin.bench.rs`

Reply:
`dev` is [a base profile](https://doc.rust-lang.org/cargo/reference/profiles.html) and is always enabled. We can re-enable LTO for it if you think it makes sense, but I don't think that it is worth it on the compile time vs. runtime balance for a local profile.

Reply:
I think currently the compile time is not that bad with dev profile, and proving code will likely need to be optimized to avoid running for a long time. So imo it would be better to re-enable lto for dev, but feel free to do as you think it would be best.

Reply:
For now, it seems that the speed is still acceptable on smallish samples even with low optimization, so I would let it be this way until we hit a wall like with MP2, where opt-level 0 was basically unusable